flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "automerge-flathubbot-prs": true,
```

Comment:
This line doesn't make any sense unless you add x-data-checker stanza to automate updates.

Reply:
My bad, honestly wasn't too aware of it, just saw other projects use it